packages/babel-plugin-proposal-import-wasm-source/test/fixtures/node-esm/import-expression-dynamic-specifier/output.mjs

```diff
@@ -0,0 +1,2 @@
+import { promises as _promises } from "fs";
```

Comment:
Would `import fs from "fs";` be better?

Reply:
Maybe? Given that this is the node-specific output both would be ok.